plugin.json

```diff
@@ -7,7 +7,7 @@
  "license": "GPL v3+",
  "homepage": "https://matomo.org",
  "require": {
-    "piwik": ">=3.1.0-rc1,<4.0.0-b1"
+    "piwik": ">=3.12.0-b3,<4.0.0-b1"
```

Comment:
@diosmosis the tests fail probably because of this change. We might need to trigger a new beta or was comparison already available in 3.12.0-b2? (I don't think so). PR looks good might just need to wait for the beta to fix the tests

Reply:
Yeah, can't merge it yet since the `getComparisons()` method doesn't exist.